cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -187,7 +184,7 @@ static Bom createBom() {
         return bom;
     }
 
-    static Bom signSBOM(String jsonFile, String pemFile) {
+    static Bom signSBOM(final String jsonFile, final String pemFile) {
```

Comment:
why? both variables are not re-assgined in singSBOM() dont think need set to final. 
or any reason to do so?